extensionsGallery-insider.json

```diff
@@ -3434,14 +3434,14 @@
 					},
 					"versions": [
 						{
-							"version": "1.0.1",
-							"lastUpdated": "4/11/2023",
+							"version": "1.1.0",
+							"lastUpdated": "5/4/2023",
```

Comment:
also please update the `Microsoft.AzDataEngine` version dependency to `1.44.0` since the move/exclude folder changes depend on the new STS changes that weren't in `1.43.0`

Reply:
this also needs to be updated in the sql project's package.json

Reply:
Done.